src/DataGrid.tsx

```diff
@@ -245,10 +245,10 @@ function DataGrid<R, SR, K extends Key>(
   const rowHeight = rawRowHeight ?? 35;
   const headerRowHeight = rawHeaderRowHeight ?? (typeof rowHeight === 'number' ? rowHeight : 35);
   const summaryRowHeight = rawSummaryRowHeight ?? (typeof rowHeight === 'number' ? rowHeight : 35);
-  const rowRenderer = renderers?.rowRenderer ?? defaultRenderers?.rowRenderer ?? defaultRowRenderer;
+  const renderRow = renderers?.renderRow ?? defaultRenderers?.renderRow ?? defaultRenderRow;
```

Comment:
May be then this should be called `renderGridRow` but probably too verbose.

Reply:
What about `renderRowWrapper`/`wrapRow`? 🤔

Reply:
May be `renderRow` is fine for now.